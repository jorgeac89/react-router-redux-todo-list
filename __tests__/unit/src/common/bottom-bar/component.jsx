import React from 'react';
import { shallow } from 'enzyme';
import BottomBar from 'common/bottom-bar';

describe('Bottom bar component', () => {
  const BottomBarComponent = BottomBar.component;

  it('renders correctly', () => {
    expect(shallow(<BottomBarComponent />)).toMatchSnapshot();
  });
});
