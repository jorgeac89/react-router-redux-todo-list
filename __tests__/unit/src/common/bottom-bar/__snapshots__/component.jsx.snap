// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottom bar component renders correctly 1`] = `
<div
  className="container-fluid bg-dark text-light mt-5"
>
  <div
    className="row py-2"
  >
    <div
      className="col-12 d-flex flex-column flex-md-row justify-content-center text-center text-nowrap"
    >
      <div>
        <span
          className="mx-3"
        >
          Jorge Antón Caballero
        </span>
        <a
          className="text-reset text-decoration-none mx-3"
          href="mailto:jorgeac89@gmail.com"
        >
          <i
            className="fas fa-envelope"
          />
            jorgeac89@gmail.com
        </a>
      </div>
      <div
        className="mt-2 mt-md-0"
      >
        <a
          className="text-reset text-decoration-none mx-3"
          href="https://github.com/jorgeac89/"
        >
          <i
            className="fab fa-github"
          />
            Github
        </a>
        <a
          className="text-reset text-decoration-none mx-3"
          href="https://www.linkedin.com/in/jorge-anton-caballero"
        >
          <i
            className="fab fa-linkedin-in"
          />
            Linkedin
        </a>
      </div>
    </div>
  </div>
</div>
`;
